Source/RSP/RSP Command.c

```diff
@@ -431,7 +432,7 @@ void RefreshRSPCommands ( void )
 	if (InRSPCommandsWindow == FALSE) { return; }
 
 	GetWindowText(hAddress,AsciiAddress,sizeof(AsciiAddress));
-	location = AsciiToHex(AsciiAddress) & ~3;

```

Comment:
`long` could be a 64-bit type.

`location = strtoul(AsciiAddress, NULL, 16) & ~3;`
If `strtoul` returns 0xFFFFFFFFFFFFFFFF, then `location = 0xFFFFFFFFFFFFFFFC`.

`location = strtoul(AsciiAddress, NULL, 16) & ~3;`
If `strtoul` returns 0xFFFFFFFFFFFFFFFF, then `location = 0x00000000FFFFFFFC`.

I forced a 32-bit result to keep consistent with your max string length limit of 8 characters in your old `AsciiToHex` function but if you don't care about maintaining that property I don't care I can just remove it.